src/app/data-planes/views/DataPlaneListView.spec.ts

```diff
@@ -46,13 +92,43 @@ describe('DataPlaneListView', () => {
     expect(tableRows.length).toBe(9)
 
     const firstTableRowHtml = tableRows[0].html()
-    const expectedTableRowStrings = ['backend', 'default', 'http', 'February 17, 2021', '1.0.7']
+    const expectedTableRowStrings = ['fake-backend', 'fake-default', 'http', 'February 18, 2021', '1.0.8']
     for (const string of expectedTableRowStrings) {
       expect(firstTableRowHtml).toContain(string)
     }
   })
 
   test('gateway listing has expected content and UI elements', async () => {
+    mock('/meshes/:mesh/dataplanes+insights', {
+      KUMA_DATAPLANE_COUNT: '2',
+    }, (merge) => {
+      return merge({
+        body: {
+          items: [
+            {
+              name: 'fake-alarm-gateway-0',
+              dataplane: {
+                networking: {
+                  gateway: {
+                    type: 'DELEGATED',
+                  },
+                },
+              },
+            },
+            {
+              name: 'fake-transmitter-gateway-0',
+              dataplane: {
+                networking: {
+                  gateway: {
+                    type: 'BUILTIN',
+                  },
+                },
+              },
+            },
+          ],
+        },
+      })
+    })
```

Comment:
Note we don't have to provide the entire response here, just the bits we are interested in.